intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -10,22 +10,303 @@ \chapter{Preparing the Environment}
 \item  Often several versions of \textbf{software packages and libraries} are installed, so you need to select the ones you need. To manage different versions efficiently, the VSC clusters use so-called \textbf{modules}, so you will need to select and load the modules that you need.
 \end{enumerate}
 
-\section{Connecting to the \hpc}
+\section{First Time connection to the \hpc}
+\label{sec:first-time-connection-to-the-hpc}
 
-To make a connection to the \hpc clusters, the ssh command is used:
+\ifwindows
+
+  \subsection{Open a Terminal}
+  \label{sec:windows-open-a-terminal}
+
+  You've generated a public/private key pair with PuTTYgen and have an approved
+  account on the VSC clusters.  The next step is to setup the connection to (one
+  of) the \hpc.
+
+  In the screenshots, we show the setup for user ``\strong{\emph{vsc20167}}''
```

Comment:
ok, but then make this very clear that we're using `vsc20167` here because it matches what's in the screenshots